cache/manager.go

```diff
@@ -1112,7 +1240,7 @@ func adaptUsageInfo(info *client.UsageInfo) filters.Adaptor {
 		case "id":
 			return info.ID, info.ID != ""
 		case "parent":
-			return info.Parent, info.Parent != ""
+			return strings.Join(info.Parents, ","), len(info.Parents) > 0
```

Comment:
Would client need to know "mergeparents" specifically here. Or can't it just assume that from the array length if needed. If we call the new field `Parents` then we can remove the other field in the future.

For filters, idk. @AkihiroSuda any ideas or do we have other places with arrays?

Reply:
Yeah I suppose it would be annoying to have to deal with two separate fields forever just to avoid a deprecation at this moment. Switched it to deprecate `Parent` and replace it with `Parents`. Also am using the `;` separated syntax now for the filter, but can easily update it there's any other ideas.